aggregator/src/main/java/com/redhat/cloud/notifications/DailyEmailAggregationJob.java

```diff
@@ -74,17 +74,22 @@ public void processDailyEmail() {
         Gauge.Timer durationTimer = duration.startTimer();
 
         try {
-            LocalDateTime now = LocalDateTime.now(UTC);
+            // get targeted schedule execution time
+            LocalDateTime now = computeScheduleExecutionTime();
```

Comment:
Why do we need to change this while the SQL that consumes the object still has the same logic? Is this fixing a bug?

Reply:
As discussed, it's to improve collected data accuracy